robottelo/ui/sync.py

```diff
@@ -129,7 +129,11 @@ def enable_rh_repos(self, repos_tree):
         :return: None.
 
         """
-        strategy, value = locators['rh.prd_expander']
+        if ostree:
```

Comment:
Here is an idea to support any type
Taken that you have next locators:
```
"rh.rpm_prd_expander": (
"rh.kickstarts_prd_expander": (
"rh.source_rpms_prd_expander": (
"rh.ostree_prd_expander": (
```
Logic:
```
def enable_rh_repos(self, repos_tree, repo_type='rpm'):
"""proper docstring explaining logic"""
locator = 'rh.{0}'_prd_expander.format(repo_type.lower())
self.click(locator)
```